ansible/group_vars/nimbus-pyrmont-stable.yml

```diff
@@ -4,3 +4,6 @@ beacon_node_cont_tag: 'stable-{{ (hostname is search("large")) | ternary("large"
 dist_validators_layout:
   "stable-large-01.aws-eu-central-1a.nimbus.pyrmont": { start:     0, end:  2500 } # 2500
   "stable-small-01.aws-eu-central-1a.nimbus.pyrmont": { start:  2500, end:  4500 } # 2000
+beacon_node_slashing_db_kind:
+  "stable-large-01.aws-eu-central-1a.nimbus.pyrmont": 'both'
+
```

Comment:
This change will not work. You can't just assume that other variables have the same format as `dist_validators_layout`.

The special format of `dist_validators_layout` is supported by a specific implementation in the `distribute-validators` role:
https://github.com/status-im/infra-nimbus/blob/86ce49aae4ddeac5348b6c4a78f189d088013ceb/ansible/roles/distribute-validators/defaults/main.yml#L13-L17